lib/pages-router/handlers/enable-draft.ts

```diff
@@ -0,0 +1,76 @@
+import { NextApiHandler, NextApiRequest, NextApiResponse } from 'next';
+import { draftMode } from 'next/headers';
+import { buildRedirectUrl, parseRequestUrl } from '../../utils/url';
+import { parseVercelJwtCookie, type VercelJwt } from '../../utils/vercelJwt';
+
+export const enableDraftApiHandler: NextApiHandler = async (
```

Comment:
We have to figure out naming here. I can't reuse the same name unless we figure out a clean way to export them from separate packages

Reply:
Yeah 🤔 I know its not pretty but `enablePagesDraftApiHandler` and `enableAppDraftApiHandler` might be the simplest for now, what do you think?